ccdproc/combiner.py

```diff
@@ -99,15 +142,12 @@ class Combiner:
                  [ 0.66666667,  0.66666667,  0.66666667,  0.66666667]]...)
     """
 
-    def __init__(self, ccd_iter, dtype=None):
+    def __init__(self, ccd_iter, dtype=None, xp=None):
```

Comment:
Not necessarily for this PR, but the validation logic in the later for-loop can be improved, e.g.

```python
ccd_list = list(ccd_iter)
if not all(isinstance(x, CCDData) for x in ccd_list):
    raise TypeError...

default_shape = ccd_list[0].shape
if not all(x.shape == default_shape):
    raise TypeError...

default_unit = ...
```